test/test_jit.py

```diff
@@ -3556,6 +3556,35 @@ def annotate_none_no_optional():
         self.checkScript(annotate_none, ())
         self.checkScript(annotate_none_no_optional, ())
 
+    @unittest.skipIf(PY2 or sys.platform == "win32", "TODO: need to fix this test case for Windows")
```

Comment:
what's the problem with windows?

Reply:
This was a leftover from `NamedTemporaryFile` which does have some issues on windows, but it's not used here so there is no problem with windows